packages/config-service/src/services/globalConfig.ts

```diff
@@ -283,43 +283,31 @@ export class GlobalConfig {
       envName: 'HBAR_RATE_LIMIT_BASIC',
       type: 'number',
       required: false,
-      defaultValue: null,
+      defaultValue: 1_120_000_000, // 11.2 hbar
```

Comment:
nit: note the assumed duration for context

Reply:
It's right below it:
<img width="996" alt="image" src="https://github.com/user-attachments/assets/974573e1-bac8-4b3a-8d7e-6f25e3f29e73" />